index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
             </nav>
         </header>
         <main>
-            <section>
+            <div class="main">
+                <section id="about-me">
+                    <div>
+                        <div>
+                            <h1>
+                                I'm William
+                                Glad to see you!
+                            </h1>
+                        </div>
+                        <div class="description">
+                            <p>I'm a software developer! I can help you build a product , feature or website Look
+                                through
+                                some
+                                of my
+                                work and
+                                experience! If you like what you see and have a project you need coded, don’t hestiate
+                                to
+                                contact
+                                me.</p>
+                        </div>
+                        <div class="connect">
+                            <h5>LET'S CONNECT</h5>
+                            <ul>
+                                <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
+                                <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
+                                <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
+                                <li><a href="#"><img src="assets/img/github.png" alt="github" /></a></li>
+                                <li><a href="#"><img src="assets/img/angies.png" alt="angies" /></a></li>
+                            </ul>
+                        </div>
+                    </div>
+                </section>
+            </div>
+            <section id="works">
+                <article class="media">
+                    <div class="media-img img-1"></div>
+                    <div class="media-body">
+                        <h4>Tonic</h4>
+                        <div class="role">
+                            <span>CANOPY</span>
+                            <i></i>
+                            <span>Backend Dev</span>
```

Comment:
I think your project is look a little bit different from the design in some aspects, kindly check out the images below.

your app
<img width="281" alt="figma-intervak" src="https://user-images.githubusercontent.com/64643995/212485343-00eef13f-993c-44db-a8bd-d4088411c9d1.png">

the design
<img width="347" alt="yoyr-interval" src="https://user-images.githubusercontent.com/64643995/212485361-8110c84e-5212-41b8-987d-b13a3beea2d0.png">

kindly implement the changes as it is a requirement that you stick to the design😉